lib/sqlalchemy/engine/result.py

```diff
@@ -461,7 +472,7 @@ def process_row(
                     processors: Optional[_ProcessorsType],
                     key_to_index: Mapping[_KeyType, int],
                     scalar_obj: Any,
-                ) -> Row[Any]:
+                ) -> Row[Unpack[_RawRowType]]:
```

Comment:
**Michael Bayer** (zzzeek) wrote:

_RawRowType should be TupleAny

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986